File: app/src/main/java/com/elegion/android/ui/widget/NoInternetView.java
package com.elegion.android.ui.widget;

import android.content.Context;
import android.util.AttributeSet;
import android.widget.Button;

import com.elegion.android.R;

import butterknife.BindView;

/**
 * Created by marat.taychinov
 */
public class NoInternetView extends EmptyView {
    @BindView(R.id.no_internet_button)
    Button mButton;

    public NoInternetView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    @Override
    protected int getLayout() {
        return R.layout.w_no_internet;
    }

    @Override
    protected Button getButton() {
        return mButton;
    }
}

File: app/src/main/kotlin/com/elegion/android/bankex/ui/widget/NoInternetView.kt
package com.elegion.android.bankex.ui.widget

import android.content.Context
import android.util.AttributeSet
import android.widget.Button

import com.elegion.android.bankex.R
import kotlinx.android.synthetic.main.w_no_internet.view.*

class NoInternetView(context: Context, attrs: AttributeSet) : EmptyView(context, attrs) {
    override fun getLayout(): Int = R.layout.w_no_internet

    override fun getButton(): Button = noInternetButton
}